docs/content/design/coordinator.md

```diff
@@ -69,34 +69,46 @@ Each run, the Druid coordinator compacts small segments abutting each other. Thi
 segments which may degrade the query performance as well as increasing the disk space usage.
 
 The coordinator first finds the segments to compact together based on the [segment search policy](#segment-search-policy).
-Once some segments are found, it launches a [compact task](../ingestion/tasks.html#compaction-task) to compact those segments.
-The maximum number of running compact tasks is `min(sum of worker capacity * slotRatio, maxSlots)`.
-Note that even though `min(sum of worker capacity * slotRatio, maxSlots)` = 0, at least one compact task is always submitted
+Once some segments are found, it launches a [compaction task](../ingestion/tasks.html#compaction-task) to compact those segments.
+The maximum number of running compaction tasks is `min(sum of worker capacity * slotRatio, maxSlots)`.
+Note that even though `min(sum of worker capacity * slotRatio, maxSlots)` = 0, at least one compaction task is always submitted
 if the compaction is enabled for a dataSource.
 See [Compaction Configuration API](../operations/api-reference.html#compaction-configuration) and [Compaction Configuration](../configuration/index.html#compaction-dynamic-configuration) to enable the compaction.
 
-Compact tasks might fail due to some reasons.
+Compaction tasks might fail due to some reasons.
```

Comment:
Suggest rewording this to "Compaction tasks might fail due to the following reasons:"